tests/sqlparser_clickhouse.rs

```diff
@@ -220,6 +220,40 @@ fn parse_create_table() {
     );
 }
 
+#[test]
+fn parse_create_view_with_fields_data_types() {
+    match clickhouse().one_statement_parses_to(
+        r#"CREATE VIEW v (i "int", f String) AS SELECT * FROM t"#,
+        r#"CREATE VIEW v (i "int", f STRING) AS SELECT * FROM t"#,
```

Comment:
Are we testing something specific about the behavior with the different casing? Thinking if not we can probably use verified_stmt with one canonical representation of the text?

Reply:
Updated with `verified_stmt`.